packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -10,21 +10,30 @@ import {
   concatAST,
 } from 'graphql';
 import chalk from 'chalk';
-import {dirname} from 'path';
+import {dirname, resolve} from 'path';
 import {readJSON, readFile, writeFile, mkdirp} from 'fs-extra';
 import {watch} from 'chokidar';
 import * as glob from 'glob';
+import {
+  getGraphQLConfig,
+  GraphQLProjectConfig,
+  GraphQLConfig,
+} from 'graphql-config';
 import {compile, Operation, Fragment, AST} from 'graphql-tool-utilities/ast';
 
 import {printDocument, printSchema} from './print';
 import {EnumFormat} from './types';
 
 export {EnumFormat};
 
-export interface Options {
+export interface CLIOptions {
+  cwd: string;
```

Comment:
this `cwd` member is a bit of a _flag_ option, but also could be useful if we wanted to add custom cwd support to the cli. basically i needed a way to short circuit the existing SK usage so that it defaulted to using the workspace `.graphqlconfig` instead of interpreting the usage as a CLI invocation.

Reply:
turns out you probably need this if you want the cli to use a config from a specific path